docs/CHANGELOG.md

```diff
@@ -4,9 +4,10 @@
 
 ## CLI
 
-## Catalog
+## Catalog, Lambdas
```

Comment:
Let's make this change in the separate commit/PR, because ideally we should this fix for 3.2.1 to stable branch.